### DISCUSSION TITLE
nginx反向代理添加认证后无法访问

### DISCUSSION THREAD

**Discussion by chuyanan** [AlistGo/alist#2758] [Q&A]
alist3我在nginx反向代理配置添加认证后，无论用户名和密码是否正确， 认证框无限重复弹出，认证配置如下：

    auth_basic           "Administrator's Area";
    auth_basic_user_file /root/.htpasswd;

**Comment by arrio464** [comment]
https://github.com/alist-org/alist/issues/3829
希望未来能够将 Authorization 修改为其他关键字
